Dockerfile

```diff
@@ -14,5 +14,8 @@ RUN addgroup --system GraphQLApp \
 USER GraphQLApp
 WORKDIR /home/GraphQLApp
 
-RUN apt-get update && apt-get install -y python3.10.5 python3.10.5-dev
+RUN apt-get update && apt-get install -y python3.10.5 python3.10.5-dev &&\
+    sudo apt-get -y install python3-pip &&\
+    pip3 install --user pipenv
```

Comment:
Also, why call the install method two times instead of calling it just once?
```
apt-get install -y python-x python-y python3-smtn \
    && pip3 install --user pipenv
```

Reply:
Moreover, we can remove this layer by simply adding all the dependencies to the first `RUN` command, this will reduce the size of the image as fewer layers will be created.